proto/ibc/applications/interchain_accounts/v1/types.proto

```diff
@@ -20,8 +20,10 @@ message IBCTxBody {
 // behalf of the interchain account.
 enum Type {
```

Comment:
Super small nit would be doc comments for these enum types but I think we can look after that with the audit